### ISSUE TITLE
Stroke sensitivity/Tapered lines

### ISSUE THREAD

**Issue by Sam15L** [federicoiosue/Omni-Notes#844]
Hey, is it possible to add pressure/speed sensitivity to the brush strokes?

While the sketch system works very well, it lacks line tapering off at the end/start of a stroke which makes it feel very flat.

Here's an example of tapered in Keep

![tapered](https://user-images.githubusercontent.com/80464997/144625249-8c21eed1-d44c-497d-979a-b8a11cb99f30.png)

And the uniformity in Omni

![uniform](https://user-images.githubusercontent.com/80464997/144625311-02832199-b589-41ce-bc9d-38e7ef244d34.png)

I think this might be the only feature (apart from a dynamic layout or redo) that Keep has over Omni Notes.

**Comment by federicoiosue** [comment]
I agree actually and I'm completely open to get any contribution on this but it's not actualy a priority compared to other features/improvements. 
So I'm not going to develop anything on that component personally. I'm going to add some specific labels to the issue.
